meson.build

```diff
@@ -408,9 +408,9 @@ endif
 specs_dir_option = get_option('specsdir')
 if specs_dir_option == ''
   search_cmds = cc.cmd_array() + ['-print-search-dirs']
-  install_dir=run_command(search_cmds).stdout().split('\n')[0]
+  install_dir=run_command(search_cmds, check : true).stdout().split('\n')[0]
   # Meson 0.56 adds a 'substring' method which can be used here
-  specs_dir=run_command(['expr', install_dir, ':', 'install: *\(.*\)']).stdout().split('\n')[0]
+  specs_dir=run_command(['expr', install_dir, ':', 'install: *\(.*\)'], check : true).stdout().split('\n')[0]
```

Comment:
Looks like this one needs to be 'false' as expr will exit with non-zero status when the result is empty, and that is checked below.

Reply:
The right thing to do would be to check the exact status code: "Exit status is 0 if EXPRESSION is neither null nor 0, 1 if EXPRESSION is null or 0, 2 if EXPRESSION is syntactically invalid, and 3 if an error occurred.". So any other exit code should be translated to an error. I hope that Meson can do that.

Reply:
Yup, looks like you can get the exit status by stashing the run_command result in a variable and looking at the .returncode value. https://mesonbuild.com/External-commands.html